src/ui/components/DevTools.js

```diff
@@ -114,6 +115,7 @@ function DevTools({
 
   return (
     <>
+      <CreateRecordingSession />
```

Comment:
hmm, this should be just a hook if we're in devtools. The child component was a hack because apollo had not been setup yet